lib/ansible/modules/crypto/acme/acme_certificate_revoke.py

```diff
@@ -38,6 +38,29 @@
     description:
       - "Path to the certificate to revoke."
     required: yes
+  account_key_src:
```

Comment:
oh, did we really miss these? and even the bot didn't notice?

Reply:
Nope, we didn't :) These two were in the doc fragment. The point I added them here as well is that here the description should be slighly different: as opposed to the other modules, where `account_key_src` and `account_key_content` are mutually exclusive, we have the 4-group `account_key_src`, `account_key_content`, `private_key_source`, `private_key_content` for this module. Before this override, the description for these two mentioned only these two, but the description for the others mentioned all four (as well as the general description).